web/components/admin/Toolbar/index.js

```diff
@@ -22,6 +29,13 @@ class Toolbar extends React.Component {
           name="collection-input"
           value={this.collectionSelectLabels[applicantType]}
           options={Object.values(this.collectionSelectLabels)} />
+        <Select
```

Comment:
Might also want to add `className="margin-left-s"` to this Select component to give it some spacing